6.X/6.9.0/docs/01-command_line_arguments.md

```diff
@@ -0,0 +1,332 @@
+`--cleanup`
+
+- If set, will attempt to automatically delete any generated diff files on the filesystem if the directory used to store the diff files already exists. Requires both `--write-diff` and `--out-diff-dir` to be set.
+
+`--secure`
+
+- applies some filters to the produced diff in order to avoid printing sensitive informations. This is useful if you need to share diffs to user who shouldn't have access to credentials.
+
+`--summary`
+
+- Only prints a summary of the diff errors.
+
+Diff command also has options inherited from the generate command options. See **generate** command section.
+
+### Sub-Command: secure-diff [DEPRECATED]
+
+Performs a diff not showing sensitive information.
+This subcommand is DEPRECATED and will be removed starting from version 6.7.0, you can have the same effect with the diff command and the flag `--secure`.
+
+## Command: `generate`
+
+`--manifest-directory`
+
+- Set the location where the installer will write the genearted manifests.
+
+`--skip-generate`
+
+- Skips generating Kubernetes manifests and attempts to diff whatever is in the manifests directory. Manifest directory can be specified using `--manifest-directory <dir>` flag.
+
+`--skip-import`
+
+- Skips the import phase, which would try to import values from a running cluster.
+
+`--skip-validation`
+
+- Skips validation checks.
+
+`--ignore-kubeconfig-errors`
+
+- This will ignore all errors from trying to parse kubeconfig file.
+
+`--preserve-templates`
+
+- Preserve directory installer templates are extracted to, this should only be used for debugging purposes
+
+`--k8s-server-version`
+
+- Sets the `kubernetesServerVersion` within values.
+
+`--helm-install`
+
+- The installer will extract the necessary files for an installation using the `helm` command only. By default it will create a directory `helm-install` in the directory where the installer is being executed. Content of the directory:
+
+  - `values.hi.yaml`: the complete values generated by the `installer`
+  - `values.hi.nats.yaml` and `values.hi.nats.global.yaml`: values for the rendering of NATSJS
+  - `charts`: the Helm charts that make up the Sysdig onprem stack
+
+`--helm-install-out-dir`
+
+- To use a custom directory to output the files generated by `--helm-install` instead of the default.
+
+### ArgoCD Generation
+
+We have introduced a way to generate ArgoCD apps definitions so that the sysdig stack can be installed using ArgoCD.
+
+At the moment we only take care of the generation of the files, the actual deploy of these files in ArgoCD is left to the user.
+
+`--argocd (boolean)`
+
+Generates files needed to deploy the sysdig stack on an ArgoCD installation. If the ArgoCD output directory exists, it will be deleted and recreated. NOTE: that using this flag will automatically also run the generation of charts that you would obtain with the `--helm-install` cli flag, this is because the argoCD generation is strictly tied with the helm-charts being produced in a certain way.
```

Comment:
rewrite it in a new paragraph: 
Using this flag will automatically generate the charts that you would obtain with the `--helm-install` CLI flag. This happens because the argoCD generation is closely linked to the specific production method of the helm-charts.